lock-api/src/main/java/com/palantir/lock/client/AsyncTimeLockUnlocker.java

```diff
@@ -56,7 +57,10 @@ public static AsyncTimeLockUnlocker create(TimelockService timelockService) {
                     } catch (Throwable t) {
                         log.info("Failed to unlock lock tokens {} from timelock. They will eventually expire on their "
                                         + "own, but if this message recurs frequently, it may be worth investigation.",
-                                SafeArg.of("lockTokens", allTokensToUnlock),
+                                SafeArg.of("numFailed", allTokensToUnlock.size()),
+                                SafeArg.of("firstFailures",
```

Comment:
because of the lock leasing, logging the request id becomes less useful, since it's a randomly client assigned id and not the actual request id that gets sent to the server, they can't do anything with this request id (for the `LockTokenShare` class at least)

this is because of us sharing the a lock for the immutable timestamp for `n` transactions

I'm somewhat biasing towards *not* logging any request ids, vs logging potentially non-actionable ones.